application/readme.md

```diff
@@ -0,0 +1,8 @@
+# Submission Design Thoughts
+
+> Ian
```

Comment:
Did you mean to include this?

Reply:
Ehhhhhh it was in the other branch that i never merged in.

It doesn't really fit this PR but it's valuable content that should probably get included anyway.

Reply:
I just worry that Ian may not want that quote publicized (even though our members understand that there's some amount of curation required in the talk selection process).

Reply:
Good point, i'll prune it